src/components/Identity/createMigration.js

```diff
@@ -10,33 +10,52 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { getApexDomain, cookieJar } from "../../utils";
+import { getApexDomain, cookieJar, isFunction } from "../../utils";
+import getVisitorECID from "./visitorService/getVisitorECID";
+import awaitVisitorOptIn from "./visitorService/awaitVisitorOptIn";
```

Comment:
Notice that when we get around to writing unit tests for createMigration.js, we'll need to inject these dependencies instead of importing them here, so we can mock them in the tests. In other words, all imports (except for utilities), will be in `Identity/index.js` and then injected down through the other Identity modules. We'll tackle that later though.